lib/driver.js

```diff
@@ -48,7 +49,7 @@ function Setup() {
         //are we running the tests on the local machine?
         if (localServer === true) {
           log('test locally');
-          if (tgtBrowser !== 'chrome' && tgtBrowser !== 'phantomjs') {
+          if (tgtBrowser !== 'chrome' && tgtBrowser !== 'phantomjs' && tgtBrowser !== 'firefox') {
```

Comment:
Nemo v2 assumes node@4 at least, which supports `Array.prototype.indexOf`. I agree that this would be a good change. Will make it as soon as I get a chance today.